terraform/bootstrap/import.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+read -p "Are you sure you want to import terraform state (y/n)? " verify
+
+if [[ $verify == "y" ]]; then
+  echo "Importing bootstrap state"
+  ./run.sh import module.s3.cloudfoundry_service_instance.bucket e6bbad10-b4f9-4c33-86e2-69ae8ce67cc4
```

Comment:
suggestion (non-blocking): Hard coding a bucket seems fragile and counter to reuse. As noted before, it might make sense to pull bootstrap out of here.